src/components/ui/Multivalue/MultivalueTag.tsx

```diff
@@ -4,19 +4,27 @@ import {MultivalueSingleValue} from '../../../interfaces/data'
 import {MultivalueFieldMeta} from '../../../interfaces/widget'
 import styles from './MultivalueTag.less'
 import cn from 'classnames'
+import {Store} from '../../../interfaces/store'
+import {connect} from 'react-redux'
+import {buildBcUrl} from '../../..'
 
-export interface MultivalueTagProps {
+export interface MultivalueTagOwnProps {
     disabled: boolean,
     placeholder?: string,
     value: MultivalueSingleValue[],
     widgetFieldMeta: MultivalueFieldMeta,
     bcName: string,
+    popupBcName: string,
```

Comment:
I think it's better to make an optional `loading?: boolean` props and leave the component unconnected while calculating the flag in [MultivalueField](https://github.com/tesler-platform/tesler-ui/blob/master/src/components/Multivalue/MultivalueField.tsx)

Reply:
fixed